### PR TITLE
add deploy.sh script

### PR DESCRIPTION
**Pull request by leewyang**
This is intentionally minimalistic and is only lightly tested, since it's hard to test outside of the CI/CD environment.  It is assumed to be running inside the CI docker container (w/ required dependencies).

### CODE REVIEW

**Comment by leewyang** [comment]
build

**Comment by leewyang** [comment]
build

File: ci/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    echo "Can only deploy from a version tag."
+    exit 1
+fi
+
+# exit on first fail
+set -e
+
+# build and publish pip package
+pushd python
+pip install -r requirements_dev.txt && pip install -e .
```
Comment (eordentlich) [inline ci/deploy.sh:15]: do you need pin install -e to build?

Comment (eordentlich) [reply ci/deploy.sh:15]: Ah makes sense.

**Review by eordentlich — APPROVED** [review]
👍

**Pull request merged**